dash-renderer/package.json

```diff
@@ -13,8 +13,8 @@
     "build:dev.watch": "webpack --mode development --watch",
     "build": "webpack --mode production",
     "start": "webpack-serve ./webpack.serve.config.js",
-    "format": "prettier --config .prettierrc --write src/**/*.js src/**/*.react.js",
-    "format:test": "prettier --config .prettierrc src/**/*.js src/**/*.react.js --list-different",
```

Comment:
Not sure really what was going on here... but from https://prettier.io/docs/en/cli.html:

> Don't forget the quotes around the globs! The quotes make sure that Prettier expands the globs rather than your shell, for cross-platform usage. The glob syntax from the glob module is used.

Anyway with quotes we don't need separate `.js` and `.react.js` patterns, and we DO match files in the root of src.

Reply:
Looks like this change needs to be propagated to all our repos in fact...